perf/benchmark/configs/istio/ext-authz/run.sh

```diff
@@ -0,0 +1,74 @@
+#!/bin/bash
```

Comment:
try to separate the run.sh into prerun.sh and postrun.sh to align with the framework? check other folder for examples

Reply:
fixed

Reply:
looks like this file has not been renamed to postrun.sh..but others lgtm